src/main/scala/org/constellation/gossip/GossipService.scala

```diff
@@ -105,14 +116,18 @@ abstract class GossipService[F[_]: Parallel, A](
       _ <- T.sleep(getPathTimeout(message.path))
       succeeded <- messageTracker.isSuccess(message)
       _ <- if (succeeded) {
-        messageTracker.remove(message.path.id) >> logger.debug(s"Succeeded path ${message.path.id}")
+        logger.debug(s"Succeeded path ${message.path.id}") >>
```

Comment:
I'm not sure if running the logic of removal before the logging and metrics isn't a better idea. If the metrics or logging fails the logic will not run I guess.

Reply:
I agree, I will revert the order.